src/components/adopters/AdopterListWrapper.js

```diff
@@ -35,6 +35,7 @@ class AdopterListWrapper extends Component {
   componentWillMount() {
     let { rows, currPage } = this.state;
     let filter = { blacklisted: false };
+    this.setState({ filter });
```

Comment:
para cuando recarga después del add to black list recargar la pagina, en el estado que estabas